### PR TITLE
Test Added for Issue 11044

### LINKED ISSUE #13881: test case for syntax error on lambdify
See this issue https://github.com/sympy/sympy/issues/11044 . That issue was fixed in PR https://github.com/sympy/sympy/pull/11566. Though no test case was added for the `lambdify`.

#### What needs to be done?

Add test case that this works just fine:
```python
>>> from sympy import MatrixSymbol, lambdify
>>> x = MatrixSymbol('x', 3, 1)
>>> f = lambdify(x, x.T*x)
```

Test case needs to be added in appropriate file

### PR DESCRIPTION
**Pull request by tanayag**
fixes #13881

### CODE REVIEW

File: sympy/utilities/tests/test_lambdify.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     symbols, lambdify, sqrt, sin, cos, tan, pi, acos, acosh, Rational,
     Float, Matrix, Lambda, Piecewise, exp, Integral, oo, I, Abs, Function,
     true, false, And, Or, Not, ITE, Min, Max, floor, diff, IndexedBase, Sum,
-    DotProduct, Eq, Dummy)
+    DotProduct, Eq, Dummy,MatrixSymbol)
```
Comment (gxyd) [inline sympy/utilities/tests/test_lambdify.py]: leave a space after `,`.

File: sympy/utilities/tests/test_lambdify.py [hunk 2]
```diff
@@ -812,3 +812,8 @@ def test_lambdify_dummy_arg():
     d2 = Dummy('x')
     f2 = lambdify(d2, d2 + 1)
     assert f2(2) == 3
+
+def test_issue_11044():
+    x = MatrixSymbol('x',3,1)
+    f = lambdify(x,x.T*x)
+    assert f(Matrix([[1],[2],[3]])) == 14
```
Comment (gxyd) [inline sympy/utilities/tests/test_lambdify.py]: add a new line here, at the end of file.

Comment (gxyd) [inline sympy/utilities/tests/test_lambdify.py]: keep a space between `[1]` and `[2]`, i.e. `Matrix([[1], [2], [3]])`

**Review by gxyd — COMMENTED** [review]
Currently code quality failure is occuring, due to absence of newline at the end of file. 
See this section for running tests locally https://github.com/sympy/sympy/wiki/Development-workflow#be-sure-that-all-tests-of-sympy-pass 

You can do `./bin/test sympy/utilities/tests/test_code_quality.py` in  your sympy root folder to run code quality tests.

**Comment by gxyd** [comment]
Please follow the default pull request template, and reference the issues by using 'Fixes #13881' in the PR description. See https://github.com/blog/1506-closing-issues-via-pull-requests.

**Comment by tanayag** [comment]
ohh cool, I am on it!

**Comment by tanayag** [comment]
I have corrected the code

File: sympy/utilities/tests/test_lambdify.py [hunk 3]
```diff
@@ -812,3 +812,9 @@ def test_lambdify_dummy_arg():
     d2 = Dummy('x')
     f2 = lambdify(d2, d2 + 1)
     assert f2(2) == 3
+
+def test_issue_11044():
+    x = MatrixSymbol('x',3,1)
+    f = lambdify(x,x.T*x)
+    assert f(Matrix([[1], [2], [3]])) == 14
+    assert f(Matrix([[2], [3], [5]])) == 38
```
Comment (gxyd) [inline sympy/utilities/tests/test_lambdify.py]: No for need for this. A single test case would do okay.

**Review by gxyd — COMMENTED** [review]
Otherwise LGTM.

**Comment by gxyd** [comment]
Sync your branch with the latest sympy master.
Also can you please squash the commits into a single commit? A lot of the commits are irrelevant in git history.
See https://github.com/sympy/sympy/wiki/Development-workflow#writing-commit-messages for how to write good commit messages.

**Comment by tanayag** [comment]
@gxyd  hey, how to squash commits into a single one??

**Comment by gxyd** [comment]
See https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git .

BTW, in future you should take for not to make changes on your 'master' branch itself.

**Comment by gxyd** [comment]
You can continue working on this for now. Let us know if get stuck at some problem with git.

**Comment by tanayag** [comment]
I dont know why the travis is failing, I guess the test case I added works fine

**Pull request closed without merging**